bin/rad-issues

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env radicle
 
-(load! "rad/prelude.rad")
-(load! "rad/monadic/issues.rad")
-(load! "rad/prelude/io-utils.rad")
+(load! (find-module-file! "rad/prelude.rad"))
```

Comment:
The `rad` prefix feels kind of redundant. Can’t we do away with it?

Reply:
Done